src/components/table/table.tsx

```diff
@@ -0,0 +1,106 @@
+import React, { ComponentPropsWithoutRef, forwardRef } from 'react';
+import cn from 'classnames';
+
+import './styles.scss';
+
+export type TableProps = ComponentPropsWithoutRef<'table'>;
+export const Table = forwardRef<HTMLTableElement, TableProps>(function Thead(
+  { className, ...props },
+  ref
+) {
+  return (
+    <table
+      ref={ref}
+      className={cn('govuk-table', 'lbh-table', className)}
+      {...props}
+    />
+  );
+});
+
+export type TheadProps = ComponentPropsWithoutRef<'thead'>;
+export const Thead = forwardRef<HTMLTableSectionElement, TheadProps>(
+  function Thead({ children, ...props }, ref) {
+    return (
+      <thead ref={ref} className="govuk-table__head" {...props}>
+        {children}
+      </thead>
+    );
+  }
+);
+
+export type TbodyProps = ComponentPropsWithoutRef<'tbody'>;
+export const Tbody = forwardRef<HTMLTableSectionElement, TbodyProps>(
+  function Tbody({ children, ...props }, ref) {
+    return (
+      <tbody ref={ref} className="govuk-table__body" {...props}>
+        {children}
+      </tbody>
+    );
+  }
+);
+
+export type TrProps = ComponentPropsWithoutRef<'tr'>;
+export const Tr = forwardRef<HTMLTableRowElement, TrProps>(function Tr(
+  { children, ...props },
+  ref
+) {
+  return (
+    <tr ref={ref} className="govuk-table__row" {...props}>
+      {children}
+    </tr>
+  );
+});
+
+export interface ThProps extends ComponentPropsWithoutRef<'th'> {
+  isNumeric?: boolean;
+}
+export const Th = forwardRef<HTMLTableHeaderCellElement, ThProps>(function Th(
+  { children, isNumeric, ...props },
```

Comment:
You're not supporting className here, you also don't need children here.

Reply:
Added className, removed children